### PR TITLE
Add task solution

### PR DESCRIPTION
**Pull request by mishakoshilka**
[DEMO LINK](https://mishakoshilka.github.io/react_person/)

### CODE REVIEW

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,19 @@
...
+
+      {person.age && (
+        <p className="Person__age">I am { person.age }</p>
+      )}
+
+      {person.isMarried ? (
+        <p className="Person__partner">{person.partnerName} is my { partnerSex }</p>
+      ) : (
+          <p className="Person__partner">I am not married</p>
+      )}
```
Comment (VitaliyBondarenko1982) [inline src/components/Person/Person.jsx]:
```suggestion
        <p className="Person__partner">
          {person.isMarried ? `${person.partnerName} is my ${partnerSex}` : 'I am not married'}
        </p>
```
enough decide only which text will be rendered.

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Good job.
Please check comment.

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const partnerSex = person.sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
```
Comment (volodymyr-soltys97) [inline src/components/Person/Person.jsx:6]: Consider will be better if you use destructuring for `person`

**Review by volodymyr-soltys97 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**